draft-packages/guidance-block/KaizenDraft/GuidanceBlock/README.mdx

```diff
@@ -34,17 +73,10 @@ The Guidance Block is a modular design element with a consistent structure and a
 
 * **Card container:** The Guidance Block content appears inside a card.
 * **Spot illustration:** Provide an illustration that assist the user in the task the Guidance Block describes.
-* **Title:** Use the title to introduce the action we want people to take.
-* **Description:** Use the description to give detail about what the user can expect about taking action.
+* **Title and description:** Use the guidelines above to guide the user with appropriate copy.
 * **Action button:** Use a link styled as a primary action button (if there are no others on the page) or a default action button to navigate to the page with the action that the Guidance Block describes. For immediate actions, show a [confirmation modal](/components/modal) instead of navigating.
 * **Optional dismiss button:** If it's possible a user will never want to take this action, consider providing a secondary dismiss button to remove the Guidance Block.
 
-### Content
-
-To supporting time-poor users scanning for actions and skim reading content, ensure the action is clear in the button text, title, and description. This includes making the subject clear as well as making it clear to the user that the Guidance Block doesn't perform an action itself, but navigates to where the action takes place. If the Guidance Block must perform an immediate action, show a [confirmation modal](/components/modal). The action indicated in the button should be clearly related to the title or description of the Guidance Block.
```

Comment:
@berry-thanki the part about the confirmation modal here was an attempt to address this feedback: https://github.com/cultureamp/kaizen-design-system/pull/390#discussion_r412567621 — what are your thoughts on resolving that use case?

Reply:
Thanks for clarification, Di! I'll add that back so it's completely clear and we're still addressing that feedback.